bloom-housing/bloom
# 1838/ami chart jurisdictionalized

## Issue Overview
Enforces Jurisdiction on the AMI charts for the partners portal

This PR addresses #issue
https://github.com/bloom-housing/bloom/issues/1838


## Description
Leverages the AMI Chart -> Jurisdiction relationship to only pull back AMI Charts on the current jurisdiction

## How Can This Be Tested/Reviewed?
The Easiest way to test this is to set some AMI charts to a different jurisdiction than the one you are on
From there ensure that when you hit the partner portal -> listing form -> units section that the AMI charts listed are only for the jurisdiction you are "on"

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
